### DISCUSSION TITLE
How dos it work ignore functions

### DISCUSSION THREAD

**Discussion by c4kaci** [dnGrep/dnGrep#526] [Q&A] [answered]
Hi,

I use 2.9.349.0 version

**1.  _"Use .gitignore"_ option**
I have .gitignore file in the target folder with this content
_node_modules/
dist/_

But I still get result from **node_modules** and **dist** folder 

**2. Same question for _"Paths to ignore"_ option**
- How can I define multiple path?
- If I define node_modules then I still get result from **node_modules**

Thx:
Karoly

**Comment by doug24** [comment, marked as answer]
1. The .gitignore file must be in a git repository, not just a file in an ordinary directory.  dnGrep runs the git command:
`git status --short --ignored`
to get the list of directories and files to ignore.  You can try that command to see the list in your git repo.

2. You can define multiple paths by separating the paths with commas or semi-colons.  Here is an example:
`*large\*;*issue*\*`
This will ignore directories with names ending with _large_ and those that contain _issue_

**Comment by c4kaci** [comment]
1. You're  right. It's only a temporary folder without .git or .github folder
2. I didn't think to asterisk character.
 
It works for me now.
Thank you!
